Documentation/doc/Documentation/Tutorials/Tutorial_triangulation_2.txt

```diff
@@ -0,0 +1,177 @@
+namespace CGAL {
+/*!
+\example Triangulation_2/Tutorial_Triangulation_2.cpp
+*/
+
+/*!
+
+\page Tutorial_Triangulation_2 2D Triangulations
+\cgalAutoToc
+
+\author Andreas Fabri
+
+A 2D triangulation is a decomposition of the 2D plane in vertices and triangular faces.
+
+In the first section we will walk you through the API of a class representing a Delaunay
+triangulation which operates on a set of points,
+followed by a section covering the API of a class representing a constrained Delaunay triangulation
+which operates on a set of points and segments.
+
+Note that this tutorial is not about computational geometry, so we assume you are familiar
+with Delaunay triangulation with and without constraints.
+
+\section Tutorial_Delaunay_2 Delaunay Triangulation
+
+In the example code of this section we use a class template `Delaunay_triangulation_2`
+together with a <em>kernel</em> which provides types for points, segments or triangles, as well as
+<em>predicates</em>, for example the incircle test needed for the empty circle property
+of this type of triangulations.
+
+So we start with some `#include` and some `using` statements to define types.
+
+\snippet Triangulation_2/Tutorial_Triangulation_2.cpp TutoT2-include
+
+When we insert a set of points in the triangulation, it generates a set of finite
+vertices corresponding to the points, and the decomposition of the convex hull
+of the points in finite triangular faces.
+Additionally, the triangulation generates infinite faces, which are incident to the edges of the convex hull
+and which have as third vertex a single infinite vertex. When we draw the triangulation
+we only draw finite vertices and faces.
```

Comment:
shouldn't you mention that the points are not inserted in the order of the range?